coinstacks/bitcoin/ingester/src/parseTx/index.ts

```diff
@@ -0,0 +1,55 @@
+import { BigNumber } from 'bignumber.js'
+import { Tx, Vin, Vout } from '@shapeshiftoss/blockbook'
+import { TxTransfer, Token } from '@shapeshiftoss/common-ingester'
+import { BTCParseTx } from '../types'
+
+// keep track of all individual tx components and add up the total value transferred
+const aggregateTransfer = (transfer: TxTransfer, value: string, token?: Token): TxTransfer => {
+  if (transfer) {
+    transfer.totalValue = new BigNumber(transfer.totalValue).plus(new BigNumber(value)).toString(10)
+    transfer.components.push({ value: value })
+  } else {
+    transfer = { totalValue: value, components: [{ value: value }] }
+  }
+
+  return { ...transfer, token }
+}
+
+export const parseTx = async (tx: Tx, address: string): Promise<BTCParseTx> => {
+  const pTx: BTCParseTx = {
+    ...tx,
+    address,
+    receive: {},
+    send: {},
+  }
+
+  // todo - assumptions:
```

Comment:
Can remove todo with addition of related issue if there is any work to do related to change address. I think we are good to go for the short term based on conversation yesterday.

Reply:
fixed